BUILD

```diff
@@ -144,6 +150,25 @@ The dinit executable will be put in /sbin (or rather, in $DESTDIR/sbin), which m
 path for normal users. Consider making a symbolic link to /usr/sbin/dinit.
 
 
+Special notes for "configure" script
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+When you using experimental "configure" script, be careful about this things:
+
+Seting environment variables in "configure" arguments: This thing works but need a backslash (\)
+with double quotation (") for every space. For example:
+
+    ./configure CXXOPTS_POST="-std=c++14\ -O3\ -g"
```

Comment:
For file paths I can accept that spaces need special treatment but it shouldn't be necessary for options like this. Please find a robust way to make this work. I think the issue currently is the `eval` on line 215; you can't just use a simple `eval` on input supplied by the user, so many things can go wrong.